GameData/Kerbalism/Profiles/Default.cfg

```diff
@@ -196,6 +213,17 @@ Profile
     dump = false
   }
 
+  // convention: 1 capacity = enough to reclaim moisture output of 1 crew member
+  Process
+  {
+    name = humidity controller
+    modifier = _HumidityController
+    input = ElectricCharge@0.037
+    input = MoistAtmosphere@0.002879122904 // 50Ltr per-day @100% humidity, contains 850 mL of Water per-day
+    output = Water@0.0000393519
```

Comment:
I'd appreciate rationales for the ratios ;-)

Reply:
Basically there is 850mL of Water in 50Ltr of saturated (100% humidity) MoistAtmosphere. Which is the same amount a Kerbal outputs in a day.
Input rate of MoistAtmosphere is calculated using:
* density of saturated Air (100% Humidity) of (8.040000e-04) (this has been multiplied by 1e3 actual real world density is 8.040000e-07 only reason I multiplied is it looked better in game otherwise input is 2.8 Units rather than 0.0028 but anyway its not so important and can be changed).
* grams/KSPUnit using density gives us 804grams/Ltr (Heavy Metal I mean Air lol).
* amount of water in a Litre of air used in calculations is 0.017mL (in real world this is approx. 100% saturation).
* 850mL of Water divided by 0.017mL = 50 Ltrs  (850mL of Water is in 50Ltrs of saturated air).
* 50Ltr divided by 804grams/Ltr = 62.18905473 Ltr of MoistAtmosphere 
* 62.18905473 Ltr / 21600 seconds (a Kerbal day of 6hrs) = 0.002879122904

Water output was even easier:
* density of water (duhh) 😛 1.0e-3 = 1000 grams/Ltr or (grams/KSPUnit)
* 850mL divided by 1000mL = 0.85 Ltr of Water
* 0.85 Ltr / 21600 seconds (a Kerbal day of 6hrs) = 0.0000393519